website/docs/reference/snapshot-properties.md

```diff
@@ -41,3 +41,44 @@ snapshots:
 
 ```
 </File>
+
+<VersionBlock firstVersion="1.9">
+
+## Define snapshots in yml
```

Comment:
would it be good to have the 'define snapshots in yaml' header visible for all versions and then have a 1.8 and lower version block telling users this is only avail for versionless and 1.9 and higher?

Reply:
Hiya @mirnawong1 

I like your suggestion too. @graciegoheen @peterallenwebb what do you think?

Kind Regards

Reply:
Sounds good to me!